ovn_k8s/common/kubernetes.py

```diff
@@ -17,22 +17,60 @@
 
 import ovs.vlog
 
+from os import path
 from ovn_k8s.common import exceptions
+from ovn_k8s.common.util import ovs_vsctl
 
+CA_CERTIFICATE = "/usr/share/openvswitch/k8s-ca.crt"
 vlog = ovs.vlog.Vlog("kubernetes")
 
 
+def _get_api_params():
```

Comment:
If we move this function to the watcher we will be forced to pass ca_certificate and token through all functions. It will make the code less readable.